Adds note for users to show which methods from GenericMap are expected to work/not work with dask arrays (#7974)
sunpy/sunpy: changelog/7974.doc.rst

> Added note for users in the methods of `~sunpy.map.GenericMap` and a table to show which methods from `~sunpy.map.GenericMap` are expected to work/not work with dask arrays.

Reviewer: ```suggestion
Added a table and notes to show which methods from `~sunpy.map.GenericMap` are expected to preserve laziness with dask arrays.
```
